packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding_email_confirmation/blocs/onboarding_email_confirmation_bloc.dart

```diff
@@ -0,0 +1,107 @@
+{{> licence.dart }}
+
+import 'package:open_mail/open_mail.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rxdart/rxdart.dart';
+
+import '../../base/common_services/count_down_service.dart';
+import '../../base/common_services/open_mail_app_service.dart';
+import '../../base/common_services/users_service.dart';
+import '../../base/extensions/error_model_extensions.dart';
+import '../../base/models/errors/error_model.dart';
+
+part 'onboarding_email_confirmation_bloc.rxb.g.dart';
+
+/// A contract class containing all events of the OnboardingEmailConfirmationBloC.
+abstract class OnboardingEmailConfirmationBlocEvents {
+  void initialize();
+
+  void startConfirmPin();
+
+  void openMailClient(String noMailAppFoundTitle);
+
+  void disableSendNewLink();
+
+  void sendNewLink();
+
+  void openMockDeepLinkSuccess();
+  void openMockDeepLinkError();
```

Comment:
Should we name these events so they contain `mock` when they will be used by others in non-mock situations? What do you think?

Reply:
They won't be used in other non-mock situations 😅 on a real device, the user will open the links through a different app, likely an email client.

These events just facilitate navigation for the 2 fake deep links presented to the user through the app itself, which would get removed in a real app. I've added the appropriate comments